srcpkgs/goreleaser/template

```diff
@@ -13,5 +13,14 @@ distfiles="https://github.com/goreleaser/goreleaser/archive/refs/tags/v$version.
 checksum=60e94d9371f7106878441deb006814339e3cf3d7f574d25cb22a9a45ec14939b
 
 post_install() {
+	vdoc README.md
```

Comment:
don't think this is useful, the only documentation-ish thing in that readme is

![image](https://github.com/void-linux/void-packages/assets/5366828/9a0d81c2-674f-4417-b48d-760fdd63920a)

which users already can see with `xbps-query -p homepage goreleaser`

Reply:
Understood. I dropped it.